nautobot/docs/administration/nautobot-server.md

```diff
@@ -480,6 +480,12 @@ Removing expired sessions...
 Invalidating cache...
 ```
 
+### `refresh_dynamic_group_member_caches`
+
+`nautobot-server refresh_dynamic_group_member_caches`
+
+Refresh the cached members of all dynamic groups. This is useful to periodically update the cached list of members of a dynamic group without having to wait for caches to expire, which defaults to one hour.
```

Comment:
```suggestion
Refresh the cached members of all Dynamic Groups. This is useful to periodically update the cached list of members of a Dynamic Group without having to wait for caches to expire, which defaults to one hour.
```

I don't see the new settings documented. If you could please add optional settings documentation for `DYNAMIC_GROUPS_MEMBER_CACHE_TIMEOUT` and `CONTENT_TYPE_CACHE_TIMEOUT` and their defaults, and then link to `DYNAMIC_GROUPS_MEMBER_CACHE_TIMEOUT` here in the "which defaults to one hour" text, that would be superfly.